app/views/submissions/_form.html.slim

```diff
@@ -210,13 +210,13 @@ javascript:
         .card-body
           - if @house_participations
             p.card-text
-              | You have earned #{@points.to_s} #{"point" + (@points == 1 ? " " : "s ")} for your house this month
+              | You have earned #{@time / 60} hr and #{@time % 60} #{"min" + (@time % 60 == 1 ? " " : "s ")} for your house this month

```

Comment:
You pluralized mins but not hrs. Might be intentional.

Reply:
removed pluralization.